lib/gakubuchi/template.rb

```diff
@@ -68,7 +68,7 @@ def assets
 
     def extract_extname(path)
       extname = path.extname
-      extname.empty? ? extname : "#{extract_extname(path.basename(extname))}#{extname}"
+      extname.empty? || extname == ".html" ? extname : "#{extract_extname(path.basename(extname))}#{extname}"
```

Comment:
__[rubocop]__
* Line is too long. [109/100] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)

<!-- Comment by [SideCI](https://sideci.com) -->